compose/backend.yml

```diff
@@ -57,7 +57,7 @@ services:
       - ccd-network
 
   ccd-data-store-api:
-    image: "${CCD_DATA_STORE_API_USE_LOCAL-hmctspublic.azurecr.io/}ccd/data-store-api:${CCD_DATA_STORE_API_TAG:-latest}"
+    image: "hmctspublic.azurecr.io/ccd/data-store-api:pr-842"
```

Comment:
why this change?

Reply:
This changes are only for our local development setup for testing and not intent to merge this PR.Sorry, for the confusion I have updated the PR description above.

Reply:
Then why raising a PR if you don't need to merge this @dharmendra8186 ? Can you close it please

Reply:
Yes, Mario, I have sorted this PR according to the comments.Actually, ccd-case-document-am-api will be update in CCD Docker once all code will be merge into master branch of our API.However, Intention of this PR to test end to end flow with dependents micro-service based on current development.